dockerfiles/frameworks/contrib/expect-phpunit-failure.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
I prefer to add expected exception instead of marking the test as skipped so if suddenly it starts to work we know immediately.